### PR TITLE
Preload Bazel 1.1.0 in the build images.

### PR DESCRIPTION
**Pull request by PiotrSikora**
Signed-off-by: Piotr Sikora <piotrsikora@google.com>

### CODE REVIEW

**Review by clarketm — APPROVED** [review]

**Comment by clarketm** [comment]
/hold
Unhold after #2042 is merged so the built image includes that change.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**
